src/test/groovy/graphql/schema/idl/SchemaPrinterTest.groovy

```diff
@@ -1,18 +1,15 @@
 package graphql.schema.idl
```

Comment:
Reverted `SchemaPrinterTest` back to original `master` version. All new comparator tests are in a new class now.